roles/ceph-rgw/tasks/multisite/main.yml

```diff
@@ -2,27 +2,57 @@
 - name: include multisite checks
   include_tasks: checks.yml
 
+- name: add realm to rgw stanza in ceph.conf
+  ini_file:
+    dest: "/etc/ceph/{{ cluster }}.conf"
+    section: "client.rgw.{{ ansible_hostname }}.{{ item.instance_name }}"
+    option: "rgw_realm"
+    value: "{{ rgw_realm }}"
+  with_items: "{{ rgw_instances }}"
+  when: rgw_instances is defined
+
+- name: add zonegroup to rgw stanza in ceph.conf
+  ini_file:
+    dest: "/etc/ceph/{{ cluster }}.conf"
+    section: "client.rgw.{{ ansible_hostname }}.{{ item.instance_name }}"
+    option: "rgw_zonegroup"
+    value: "{{ rgw_zonegroup }}"
+  with_items: "{{ rgw_instances }}"
+  when: rgw_instances is defined
+
+- name: add zonegroup to rgw stanza in ceph.conf
+  ini_file:
+    dest: "/etc/ceph/{{ cluster }}.conf"
+    section: "client.rgw.{{ ansible_hostname }}.{{ item.instance_name }}"
+    option: "rgw_zone"
+    value: "{{ rgw_zone }}"
+  with_items: "{{ rgw_instances }}"
+  when: rgw_instances is defined
```

Comment:
I'm not sure to understand why you're using the `ini_file` module here instead of modify the ceph.conf template instead.

This means that the configuration will be erase by the ceph-config role and then restore during the multisite tasks. So this will trigger the rgw handlers for nothing.

Reply:
I'll go ahead and do that. This was the way it was before just in a different part of the file.